util/automation.py

```diff
@@ -482,21 +446,15 @@ def exec_wer_run():
 
     # Collect expected checkpoint directory depending on what the user wants
     # defaulting to re-using checkpoint from the same SHA1
-    ckpt_dir, force_ckpt_restore = ensure_checkpoint_directory()
+    ckpt_dir, _ = ensure_checkpoint_directory()
```

Comment:
We can remove the second parameter, since we removed it from the return statement of ensure_checkpoint_directory